Dockerfile

```diff
@@ -106,7 +153,7 @@ RUN rm -f /tmp/resources/overlays/README.txt && \
     fi;
 
 # Optionally remove Tomcat manager, docs, and examples
-ARG TOMCAT_EXTRAS=true
+#ARG TOMCAT_EXTRAS=true #moved to docker-compose.yml
 RUN if [ "$TOMCAT_EXTRAS" = false ]; then \
```

Comment:
because the previous line has been moved to docker-compose, this will always be false and therefore run - delete it? Or should it run?

Reply:
It should always be true. The arguments are substituted when building the image. So in the docker-compose, it would be 
```
    args:
      - ORACLE_JDK=true
      - TOMCAT_EXTRAS=false
```